shunting-yard-receiver/shunting-yard-receiver-sqs/src/main/java/com/hotels/shunting/yard/receiver/sqs/messaging/SqsMessageReaderFactory.java

```diff
@@ -15,17 +15,36 @@
  */
 package com.hotels.shunting.yard.receiver.sqs.messaging;
 
+import static com.hotels.shunting.yard.receiver.sqs.SqsReceiverUtils.credentials;
+import static com.hotels.shunting.yard.receiver.sqs.SqsReceiverUtils.maxMessages;
+import static com.hotels.shunting.yard.receiver.sqs.SqsReceiverUtils.queue;
+import static com.hotels.shunting.yard.receiver.sqs.SqsReceiverUtils.region;
+import static com.hotels.shunting.yard.receiver.sqs.SqsReceiverUtils.waitTimeSeconds;
+
 import org.apache.hadoop.conf.Configuration;
 
-import com.hotels.shunting.yard.common.io.jackson.ApiarySqsMessageDeserializer;
-import com.hotels.shunting.yard.common.messaging.MessageReader;
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.AmazonSQSClientBuilder;
+
+import com.expedia.apiary.extensions.receiver.common.messaging.MessageReader;
+import com.expedia.apiary.extensions.receiver.sqs.messaging.SqsMessageReader;
+
 import com.hotels.shunting.yard.common.messaging.MessageReaderFactory;
 
 public class SqsMessageReaderFactory implements MessageReaderFactory {
```

Comment:
This looks "common" - isn't it? Just wondering why this wasn't pulled out?

Reply:
Shunting Yard here uses the hadoop `Configuration` to supply the required queue, max messages, wait time config etc. I didn't think this would be common as Hkaas would load these from a spring config file, for example. Does that make sense?

Reply:
OK, sure, that's fine. Let's leave it here then, if later on we see more commonality across different modules we can extract things further.

Reply:
Actually this feature to read credentials and region from configuration is not used. I am thinking should we remove this and then we don't need the shunting-yard-receiver and receiver-sqs anymore.